### PR TITLE
[vllm_omni, ci, doc] feat: upgrade to v0.24 and pin vllm-omni to fe478a95

### PR DESCRIPTION
**Pull request by SamitHuang**
### What does this PR do?

  - Inherit [#243](https://github.com/verl-project/verl-omni/pull/243) (`Align for vllm and vllm-omni 0.24.0`): bump `vllm` / `vllm-omni` to `0.24.0`, and default registered rollout
  pipelines to `supports_request_batch = False` (no request-batch adaptation in this PR).
  - Pin vLLM-Omni to `fe478a95ab39e80085da06e3401c3ed471898a57` ([BugFix] Restore `vllm_c` IR op priority and `torch.nn.RMSNorm` for Qwen-Image) in `.github/vllm_omni_pin.txt`.
  - Upgrade GPU CI image to `verl-ci-cn-beijing.cr.volces.com/verlai/verl:vllm023.dev1`, align `cpu_unit_tests` with `vllm==0.24.0` + the same git pin, and update `docs/start/install.md`.

  Request-level batching (`supports_request_batch = True` + batch forward adapters) is intentionally deferred to a follow-up PR.

### Checklist Before Starting

- [ ] Search for similar PRs. Paste at least one query link here: ...
- [ ] Format the PR title as `[{modules}] {type}: {description}` (This will be checked by the CI)
  - `{modules}` include `fsdp`, `vllm_omni`, `rollout`, `trainer`, `ci`, `training_utils`, `recipe`, `ray`, `worker`, `single_controller`, `misc`, `perf`, `model`, `algo`, `env`, `tool`, `ckpt`, `doc`, `data`, `cfg`, `reward`, `diffusion`, `omni`, `tests`, `docker`
  - If this PR involves multiple modules, separate them with `,` like `[diffusion, doc]`
  - `{type}` is in `feat`, `fix`, `refactor`, `chore`, `test`
  - If this PR breaks any API (CLI arguments, config, function signature, etc.), add `[BREAKING]` to the beginning of the title.
  - Example: `[BREAKING][diffusion, fsdp] feat: new rollout scheduler`

### Test

  - [x] `pre-commit run --files` on touched paths
  - [ ] CPU: `pytest -s -x --asyncio-mode=auto tests/` (or CI `cpu_unit_tests`)
  - [ ] GPU smoke CI on `vllm023.dev1` with git-pinned vllm-omni
  - [x] Local e2e Qwen-Image FlowGRPO smoke with vllm-omni `@fe478a95`

* Time Performance is similar to vllm-omni 0.22

image

* Convergence
image

### API and Usage Example

> Demonstrate how the API changes if any, and provide usage example(s) if possible.

```python
# Add code snippet or script demonstrating how to use this
```

### Design & Code Changes

> Demonstrate the high-level design if this PR is complex, and list the specific changes.

### Checklist Before Submitting

> [!IMPORTANT]
> Please check all the following items before requesting a review, otherwise the reviewer might deprioritize this PR for review.

- [ ] Read the [Contribute Guide](https://github.com/verl-project/verl-omni/blob/main/CONTRIBUTING.md).
- [ ] Apply [pre-commit checks](https://github.com/verl-project/verl-omni/blob/main/CONTRIBUTING.md#code-linting-and-formatting): `pre-commit install && pre-commit run --all-files --show-diff-on-failure --color=always`
- [ ] Add / Update the documentation.
- [ ] Add unit or end-to-end test(s) to [the CI workflow](https://github.com/verl-project/verl-omni/tree/main/.github/workflows) to cover all the code. If not feasible, explain why: ...

### CODE REVIEW

File: verl_omni/pipelines/model_base.py [hunk 1]
```diff
@@ -250,6 +250,8 @@ def register(cls, architecture: str, algorithm: str):
         """Class decorator that registers a pipeline for ``(architecture, algorithm)``."""
 
         def decorator(subclass: type) -> type:
+            if "supports_request_batch" not in subclass.__dict__:
+                subclass.supports_request_batch = False
```
[the remark this answers is not in the document]
Comment (SamitHuang) [reply verl_omni/pipelines/model_base.py:254]: Registered pipelines look like QwenImagePipelineWithLogProb(QwenImagePipeline), and upstream QwenImagePipeline already has supports_request_batch = True. Checking __dict__ is
  deliberate: only an explicit class-body opt-in counts; inherited True is ignored and forced to False. That matches this PR’s “no request-batch adaptation” choice.

**Review by zhtmike — APPROVED** [review]

**Comment by SamitHuang** [comment]
npu vllm-ascend pin is referred from https://github.com/vllm-project/vllm-omni/issues/2223#issuecomment-4705062827

@panshaowu PTAL
